terraform/distribution/main.tf

```diff
@@ -96,6 +96,7 @@ resource "aws_cloudfront_distribution" "distribution_prisma_studio" {
   viewer_certificate {
     acm_certificate_arn = data.aws_acm_certificate.certificate.arn
     ssl_support_method  = "sni-only"
+    minimum_protocol_version = "TLSv1.2_2018"
```

Comment:
This one is a freebie, Security asked us if we can turn off TLS or SSL versions 1.0 and 1.1 from our https configurations, since they are now deemed insecure.